src/main/java/dev/akarah/cdata/script/dsl/DslTokenizer.java

```diff
@@ -189,4 +181,14 @@ public String readPath() {
         }
         return this.stringReader.getString().substring(start, this.stringReader.getCursor());
     }
+
+    public DataResult<DslToken> tokenWithEquals(char symbol, Supplier<DslToken> token, Supplier<DslToken> tokenWithEquals) throws CommandSyntaxException {
```

Comment:
I decided to split this out since 4 is already a lot, and it's definitely possible there could be more [e.g. *=]